src/xrpld/app/misc/WasmHostFuncImpl.cpp

```diff
@@ -247,11 +269,11 @@ WasmHostFunctionsImpl::getTxNestedField(Slice const& locator)
 Expected<Bytes, HostFunctionError>
 WasmHostFunctionsImpl::getCurrentLedgerObjNestedField(Slice const& locator)
 {
-    auto const sle = ctx.view().read(leKey);
-    if (!sle)
-        return Unexpected(HostFunctionError::LEDGER_OBJ_NOT_FOUND);
+    auto const sle = getCurrentLedgerObj();
+    if (!sle.has_value())
```

Comment:
just `!sle`

Reply:
I'd prefer being more explicit - is there a problem with doing it this way?

Reply:
When it is used once - it is more clear, when it is used continuously feels just unnecessary.